pluma/schema/outdoor.py

```diff
@@ -25,6 +27,29 @@ def build_schema(root: Union[str, ComplexPath],
     """
     root = ensure_complexpath(root)
     streams = DotMap()
+
+    # Pupil streams
+    streams.PupilLabs.Counter.DecodedFrames =     HarpStream(209, device='PupilLabs', streamlabel='Counter_DecodedFrames', root=root, autoload=autoload, parent_dataset=parent_dataset)
+    streams.PupilLabs.Counter.RawFrames =         HarpStream(210, device='PupilLabs', streamlabel='Counter_RawFrames', root=root, autoload=autoload, parent_dataset=parent_dataset)
+    streams.PupilLabs.Data.RawFrames =            PupilStream(['PupilLabs/WorldCamera_Frame0.bin', 'PupilLabs/WorldCamera_Frame1.bin', 'PupilLabs/WorldCamera_Frame2.bin'],
```

Comment:
I would somehow prefer it if we don't have to write the file names in the schema, as it can allow for variation in the data logging format.

Would it work for pupil streams to assume we know the name of the files given the name of the stream? Another option would be to create quick sub-classes for each specialized stream.

The reason why I think this is a better approach is that we might have different experimental schemas for different experiments with different hardware configurations and so these might end up being duplicated too often. Reducing the number of free schema parameters would help ensure consistency in that case.